README.md

```diff
@@ -41,6 +41,11 @@ AWS_SQS               | AWS SQS URL
 AWS_SECRET_ACCESS_KEY | AWS Secret Access Key
 AWS_ACCESS_KEY_ID     | AWS Access Key ID
 WRITE_TO_CASEWORK     | Enable/Disable sending the case to iCasework (defaults to false)
+AUDIT_DATA            | Enable/Disable sending audit data to postgres (defaults to false)
+DB_HOST               | Postgres host for audit data
```

Comment:
Could we disambiguate these environment variables, to avoid confusion with any  "main"database we add in the future?
```suggestion
AUDIT_DB_HOST               | Postgres host for audit data
```

Reply:
@zuzak this work doesn't have a ticket AFAIK. If someone wants to create one i'm happy to reference it in the commit or PR.

Happy to update the PR based on AUDIT_* env vars

Reply:
If you've got access, could you create a ticket yourself?
You've got more context than I for creating the ticket; and it's probably good practice moving forward.